topic/timeseries/time-series-decomposition.ipynb

```diff
@@ -0,0 +1,1352 @@
+    "\n",
+    "\n",
+    "## Components\n",
+    "\n",
+    "**Seasonality**: The seasonal component captures regular patterns that repeat over fixed periods, such as daily, monthly, or quarterly fluctuations. Seasonality is influenced by various factors like weather, holidays, and business cycles.\n",
+    "\n",
+    "**Trend**: The trend component represents the long-term progression of the series, showing how the data moves up or down over a lengthy period. It's essentially the underlying direction in which the data is heading, excluding shorter-term fluctuations.\n",
+    "\n",
+    "**Residual (Irregular Component)**: After the trend and seasonal components are removed from the original time series, the remainder is the residual component. It consists of irregular or random fluctuations that cannot be attributed to the trend or seasonal effects. This component can sometimes reveal unusual events or outliers not explained by the trend or seasonality.\n",
+    "\n",
+    "\n",
+    "## What is PyCaret?\n",
+    "\n",
+    "\n",
+    "PyCaret is an open-source, low-code machine learning (ML) library in Python that aims to reduce the cycle time from hypothesis to insights. PyCaret automates much of the ML workflow, including data preprocessing, feature engineering, model selection, hyperparameter tuning, and model deployment, making it accessible for beginners while still being powerful enough for experts.\n",
+    "\n",
+    "Integrating PyCaret with CrateDB presents a compelling opportunity for handling large-scale data analytics and ML projects. CrateDB is a distributed SQL database that excels in handling massive amounts of structured and unstructured data in real-time. This integration allows users to leverage CrateDB's efficient data storage and fast query capabilities to manage large datasets, while PyCaret's ML algorithms can be applied directly to this data for predictive analytics, anomaly detection, and other advanced analytics tasks.\n",
+    "\n",
+    "By following this notebook you will learn how to extract data from CrateDB for analysis in PyCaret, how to further prerocess it and how to use  PyCaret to plot time series decomposition by breaking it down into its basic components: **trend, seasonality, and residual (or irregular) fluctuations**."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0ab53421",
+   "metadata": {},
+   "source": [
+    "## Getting started\n",
+    "\n",
+    "To get started with this notebook install necessary dependecies and import required modules:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "69f4989b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#!pip install -r requirements.txt\n",
+    "\n",
+    "# Note: If you are running in an environment like Google Colab, please use the absolute path of the requirements:\n",
+    "#!pip install -r https://raw.githubusercontent.com/crate/cratedb-examples/main/topic/timeseries/requirements.txt\""
```

Comment:
```suggestion
    "#!pip install -r https://raw.githubusercontent.com/crate/cratedb-examples/main/topic/timeseries/requirements.txt"
```

Need to remove the double quote at the end of the requirements file with full path